x/nodes/keeper/reward.go

```diff
@@ -25,7 +25,11 @@ func (k Keeper) RewardForRelays(ctx sdk.Ctx, relays sdk.BigInt, address sdk.Addr
 	}
 
 	if isAfterNonCustodial {
```

Comment:
This outer if structure can be removed

Reply:
This can be rolled back with the feature map

Reply:
So we can't remove it without losing that protection